pkg/core/interop/context.go

```diff
@@ -135,7 +135,16 @@ func NewContractMD(name string, id int32) *ContractMD {
 
 // AddMethod adds new method to a native contract.
 func (c *ContractMD) AddMethod(md *MethodAndPrice, desc *manifest.Method) {
-	c.Manifest.ABI.Methods = append(c.Manifest.ABI.Methods, *desc)
+	index := sort.Search(len(c.Manifest.ABI.Methods), func(i int) bool {
+		md := c.Manifest.ABI.Methods[i]
+		if md.Name != desc.Name {
+			return md.Name >= desc.Name
+		}
+		return len(md.Parameters) > len(desc.Parameters)
+	})
+	c.Manifest.ABI.Methods = append(c.Manifest.ABI.Methods, manifest.Method{})
+	copy(c.Manifest.ABI.Methods[index+1:], c.Manifest.ABI.Methods[index:])
+	c.Manifest.ABI.Methods[index] = *desc
```

Comment:
Set `desc.Safe` firstly (it is done 2 lines below), and after that do `c.Manifest.ABI.Methods[index] = *desc`.

Reply:
Fixed, actually this bug was already present.